statsmodels/tsa/arima_model.py

```diff
@@ -284,7 +284,7 @@ def _get_predict_out_of_sample(endog, p, q, k_trend, k_exog, start, errors,
             mu = np.array([mu]*steps)
     elif k_exog > 0:
         X = np.dot(exog, exparams)
-        #NOTE: you should not have to give in-sample exog!
+        # NOTE: you should not have to give in-sample exog!
```

Comment:
exclamation suggests this might be a TODO?

Reply:
Looks meaningless.